docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -69,9 +69,7 @@ Options `cache`, `cache-stack`, and `cache-config-file` are no longer build opti
 This eliminates the need to execute the build phase and rebuild your image due to them.
 Be aware that they will not be recognized during the `build` phase, so you need to remove them.
 
-= kcadm Changes
-
-How kcadm parses and handles options and parameters has changed. Error messages from usage errors, the wrong option or parameter, may be slightly different than previous versions. Also usage errors will have an exit code of 2 instead of 1.
+How kcadm and kcreg parse and handle options and parameters has changed. Error messages from usage errors, the wrong option or parameter, may be slightly different than previous versions. Also usage errors will have an exit code of 2 instead of 1.
```

Comment:
@shawkins Can you please (re-)add a heading here?

Reply:
Sorry, added it back.